.devcontainer/Dockerfile

```diff
@@ -0,0 +1,16 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.192.0/containers/javascript-node/.devcontainer/base.Dockerfile
```

Comment:
This approach, devcontainers/codespaces deviates from the guiding principle that moves us towards dev/prod parity. From a maintainability perspective, it's code that we'll have to maintain that is only applicable to one deployment target.

I would advise an alternative approach to creating a development environment that moves us towards free, dev/prod parity and is supported by the organization this code lives in.

Reply:
This was put in to solve a problem with the M1 Macs, which the OCIO are about to receive themselves. We very open to other suggestions here, what is your preference in terms of solutions to this problem?

Reply:
when `--platform=linux/amd64` was added to the `src/frontend/Dockerfile` 3 months ago, was that not done to address the M1 architecture issue?